kansha/board/boardconfig.py

```diff
@@ -7,17 +7,19 @@
 # the file LICENSE.txt, which you should have received as part of
 # this distribution.
 #--
-
 from collections import namedtuple, OrderedDict
-from nagare import component, security, var
+
+from nagare import component
+from nagare import editor
+from nagare import i18n
+from nagare import security
+from nagare import validator, var
 from nagare.i18n import _
 
 from kansha import notifications
 from kansha.menu import MenuEntry
-from ..label import comp as label
-from nagare import editor
-from nagare import validator
-from nagare import i18n
+from kansha.label import comp as label
```

Comment:
Don't import the logic part (`comp`) of a component. The `__init__.py` of a component must be used to export its public API (a subset of the API in its `comp`). For example, in the `__init__.py` of `kansha.title`:
```python
from .comp import Title, EditableTitle

from . import view
```
and then, here:
```python
from kansha.title import Title
```
or
```python
from kasha import title
...
my_title = title.Title(...)
```